src/css/custom.scss

```diff
@@ -202,6 +202,9 @@ img[src*="md-width"] {
     font-style: italic;
 }
 
+.newline-snippet {
+    margin: 0px !important;
+}
 .inline_svg {
```

Comment:
Just a comment that the `inline_svg` class selector is not nested under the `newline-snippet` selector. I’m not sure if it is intended to be nested, but the indentation of the former caused me to think it _could_ have been the intention.